.github/workflows/module-dockerregistry.yml

```diff
@@ -0,0 +1,56 @@
+name: DockerRegistry module pipeline
```

Comment:
Please take a look at #1414: this module file has been removed in favour of an array of modules in the `ci.yml` workflow.